tests/test_libraries.py

```diff
@@ -2150,3 +2150,17 @@ def test_itk(pyi_builder):
     pyi_builder.test_source("""
         import itk
     """)
+
+
+@importorskip('slixmpp')
+def test_slixmpp(pyi_builder):
+    pyi_builder.test_source("""
+        import slixmpp
+
+
+        class JabberClient(slixmpp.ClientXMPP):
```

Comment:
Why not instantiate `slixmpp.ClientXMPP('username', 'password')` directly?

Reply:
This is how it gets used in real code, base class doesn't do much without some methods overwritten.
For this specific test it does make no difference, I suppose.